test/e2e-tests.sh

```diff
@@ -15,12 +15,12 @@ dump_state.setup # test
 if [[ $MESH == "true" ]]; then
   # net-istio does not use knative-serving-ingress namespace.
   export INGRESS_NAMESPACE="knative-serving"
-else
-  trust_router_ca
 fi
 
 logger.success '🚀 Cluster prepared for testing.'
 
+trust_router_ca
```

Comment:
we always need to trust the CA, also for mesh with TLS (not yet available downstream).